docs/guides/hybrid-models.md

```diff
@@ -141,7 +141,7 @@ For inference with the `HybridFHEModel` instance, `hybrid_model`, call the regul
 <!--pytest-codeblocks:skip-->
 
 ```python
-hybrid_model.forward(torch.randn((dim, )))
+hybrid_model(torch.randn((dim, )))
```

Comment:
here, do we call the quantized module or really the simulate/fhe function?

Reply:
We call the quantized private modules, the modes are: simulate, disable, execute and calibrate.

 or really the simulate/fhe function?

Reply:
I thought that the simulate/fhe functions were used when we called `run`, which was below `forward` in the non-hybrid models, and that calling simply without the `run` was using the "all-in-CML (ie, no CP, no simulation nor FHE" quantized module function. But I may be wrong, to be checked with the rest of ML team maybe

Reply:
TBH, I don't know

Reply:
> here, do we call the quantized module or really the simulate/fhe function?

By default, it's the quantized module. I think it's better to add the forward in the hybrid model as said in this issue https://github.com/zama-ai/concrete-ml-internal/issues/4579.